core/src/main/java/org/elasticsearch/common/logging/log4j/LogConfigurator.java

```diff
@@ -84,6 +85,10 @@
             .immutableMap();
 
     public static void configure(Settings settings) {
```

Comment:
Maybe just update all callers to pass true for `resolveConfig`? I have a love/hate relationship with these methods with default values for positional parameters and the crummy way you have to do it in Java and figure unless there are a ton of callers we should avoid making more. But, you know, if there are tons of callers I'm super ok with it.